### ISSUE TITLE
Why lr *100 when load pretrained model?

### ISSUE THREAD

**Issue by Dylanself123** [Tencent/MedicalNet#26]
Thank you for your work. 

I wonder why lr*100 and i have found the rate of convergence could get higher with original lr. Thanks a lot.

{ 'params': parameters['base_parameters'], 'lr': sets.learning_rate },
                    { 'params': parameters['new_parameters'], 'lr': sets.learning_rate*100 }

**Comment by Ahgni** [comment]
When I read your paper, there is a sentence that I don't understand:
**When training with pre-trained models, we optimize the model parameters with Adam [41] starting from the 0.001 learning rate, while the learning rate of TFS is set to 0.01.**
So my question is why the learning rate setting is different from pre-trained models and train from scratch?

**Comment by pyjnqd** [comment]
I think TFS should need more larger learning rate for optimization convenience because it starts from origin
